### PR TITLE
Add endpoints for graph property procs

### PR DESCRIPTION
**Pull request by soerenreichardt**

### CODE REVIEW

**Comment by soerenreichardt** [comment]
[[Arrow] Graph Properties continued](https://trello.com/c/xjmRlN7M/6052-arrow-graph-properties-continued)

File: graphdatascience/graph/graph_entity_ops_runner.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def _handle_properties(
         return self._query_runner.run_query(query, params)
 
 
-class GraphPropertyRunner(GraphEntityOpsBaseRunner):
+class GraphElementPropertyRunner(GraphEntityOpsBaseRunner):
```
Comment (jjaderberg) [inline graphdatascience/graph/graph_entity_ops_runner.py:39]: is element and entity the same thing here?
its a small detail but maybe we could use the same term (if indeed it refers to the same thing)

File: graphdatascience/tests/unit/test_graph_ops.py [hunk 2]
```diff
@@ -508,6 +508,24 @@ def test_graph_relationships_drop(runner: CollectingQueryRunner, gds: GraphDataS
     assert runner.last_params() == {"graph_name": "g", "relationship_type": "REL_A"}
 
 
+@pytest.mark.parametrize("server_version", [ServerVersion(2, 2, 0)])
+def test_graph_property_stream(runner: CollectingQueryRunner, gds: GraphDataScience) -> None:
+    G, _ = gds.graph.project("g", "*", "*")
+
+    gds.alpha.graph.graphProperty.stream(G, "prop")
+    assert runner.last_query() == "CALL gds.alpha.graph.graphProperty.stream($graph_name, $graph_property)"
+    assert runner.last_params() == {"graph_name": "g", "graph_property": "prop"}
```
Comment (jjaderberg) [inline graphdatascience/tests/unit/test_graph_ops.py:517]: missing the config here?

File: graphdatascience/tests/unit/test_graph_ops.py [hunk 3]
```diff
@@ -508,6 +508,24 @@ def test_graph_relationships_drop(runner: CollectingQueryRunner, gds: GraphDataS
...
+
+
+@pytest.mark.parametrize("server_version", [ServerVersion(2, 2, 0)])
+def test_graph_property_drop(runner: CollectingQueryRunner, gds: GraphDataScience) -> None:
+    G, _ = gds.graph.project("g", "*", "*")
+
+    gds.alpha.graph.graphProperty.stream(G, "prop")
```
Comment (jjaderberg) [inline graphdatascience/tests/unit/test_graph_ops.py:524]:
```suggestion
    gds.alpha.graph.graphProperty.drop(G, "prop")
```

**Review by jjaderberg — CHANGES_REQUESTED** [review]
As far as I understand it this change is good but the tests are a bit broken

**Pull request merged**